code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -32,9 +32,21 @@
 	gold_core_spawnable = FRIENDLY_SPAWN
 	obj_damage = 0
 	environment_smash = ENVIRONMENT_SMASH_NONE
+<<<<<<< refs/remotes/BeeStation/master
+<<<<<<< refs/remotes/BeeStation/master
 	chat_color = "#26F55A"
+=======
+	mobsay_color = "#26F55A"
+>>>>>>> update
+=======
+	chat_color = "#26F55A"
+>>>>>>> update
```

Comment:
```suggestion
	chat_color = "#26F55A"
```

Reply:
dunno what you are talking about... that was never there
i totally did not miss it

Reply:
Left overs from the merge conflicts